packages/expo/test/index.test.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment jsdom
+ */
+
```

Comment:
is this the right environment for expo?

Reply:
no it's not! The issue was the network breadcrumbs plugin, I've updated the test to mock that out instead